src/api/users/src/routes/users.js

```diff
@@ -16,7 +16,7 @@ router.get('/:id', validateId(), async (req, res, next) => {
     const doc = await userRef.get();
 
     if (!doc.exists) {
-      next(createError(404, `user ${id}) not be found.`));
+      next(createError(404, `user ${id}) not found.`));
```

Comment:
Is the extra parenthesis intended here? It's repeated a few times below too.

> `user ${id}) <---

Reply:
No, good grief, let me fix.  And NO, I DON'T USE COPY/PASTE, HOW COULD YOU SUGGEST SUCH A THING!!!???

Thanks for catching my terrible code.